tests/tck/features/match/PipeAndVariable.feature

```diff
@@ -1,11 +1,142 @@
+      $v1 = FETCH PROP ON player "Tony Parker"
+      YIELD properties(Vertex) AS props, properties(Vertex).name AS name, properties(Vertex).age AS age
+      """
+    Then the result should be, in any order, with relax comparison:
+      | props                          | name          | age |
+      | {age: 36, name: "Tony Parker"} | "Tony Parker" | 36  |
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      $v2 = YIELD "Tim Duncan" AS b;
+      GO FROM $v1.a OVER like WHERE $v1.a > $v2.b YIELD id($$) AS dst;
+      """
+    Then a SemanticError should be raised at runtime: Multiple variable property is not supported in WHERE or YIELD
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      GO FROM "Tim Duncan" OVER like WHERE id($$) != $v1.a
+      YIELD id($$) AS dst
+      """
+    Then a SemanticError should be raised at runtime: A variable must be referred in FROM before used in WHERE or YIELD
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      GO FROM "Tim Duncan" OVER like WHERE id($$) != $v1.a
+      YIELD id($$) AS dst
+      """
+    Then a SemanticError should be raised at runtime: A variable must be referred in FROM before used in WHERE or YIELD
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      GO FROM "Tim Duncan" OVER like
+      YIELD id($$) AS dst, $v1.a AS dst2
+      """
+    Then a SemanticError should be raised at runtime: A variable must be referred in FROM before used in WHERE or YIELD
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      GO FROM $v1.a OVER like WHERE id($$) != $v1.a
+      YIELD id($$) AS dst, $v1.a AS dst2
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst                 | dst2          |
+      | "LaMarcus Aldridge" | "Tony Parker" |
+      | "Manu Ginobili"     | "Tony Parker" |
+      | "Tim Duncan"        | "Tony Parker" |
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      $v1 = YIELD "Tim Duncan" AS a;
+      """
+    Then a SemanticError should be raised at runtime: Variable `v1' already exists
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      $v2 = YIELD $v1.a AS a;
+      """
+    Then the result should be, in any order, with relax comparison:
+      | a             |
+      | "Tony Parker" |
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      $v2 = YIELD $v1.a AS a;
+      YIELD $v1.a AS a UNION ALL YIELD $v2.a AS a
+      """
+    Then the result should be, in any order, with relax comparison:
+      | a             |
+      | "Tony Parker" |
+      | "Tony Parker" |
+    When executing query:
+      """
+      $v1 = YIELD "Tony Parker" AS a;
+      $v2 = YIELD $v1.a AS a;
+      YIELD $v1.a, $v2.a
+      """
+    Then a SemanticError should be raised at runtime: Only one variable allowed to use.
```

Comment:
`Redefined variable: <var>`

Reply:
Redefined?

Reply:
Yes.

Reply:
add what varialbe is redefined in the error message

Reply:
I don't get it. This test case is not a redefined variable error, but multiple variable referenced in yield clause.

The case of redefined variable  is:
```
    When executing query:
      """
      $v1 = YIELD "Tony Parker" AS a;
      $v1 = YIELD "Tim Duncan" AS a;
      """
    Then a SemanticError should be raised at runtime: Variable `v1' already exists
```

Reply:
Ok, my mistake.